_2020/data-wrangling.md

```diff
@@ -253,8 +258,8 @@ ssh myserver journalctl
  | awk '{print $2}' | paste -sd,
 ```
 
-Let's start with `paste`: it lets you combine lines (`-s`) by a given
-single-character delimiter (`-d`). But what's this `awk` business?
+Let's start with `paste`: it lets you combine lines (`-s`) separated by a given
```

Comment:
Ah, so, this isn't quite accurate. The way it was is correct — `paste` lets you "combine lines ... by a given delimiter", it does not require those lines to be separated by the given delimiter.